.github/CODEOWNERS

```diff
@@ -1,55 +1,102 @@
+* @elastic/elastic-agent-data-plane
+
+
+### Private directories
+# Any changes to the CI system or .github must be reviewed by the data plane team.
+/.ci/ @elastic/elastic-agent-data-plane
+/.github/ @elastic/elastic-agent-data-plane
+
+### Auditbeat
+/auditbeat/ @elastic/security-external-integrations
 /x-pack/auditbeat/ @elastic/security-external-integrations
 
-# Packetbeat
-/packetbeat/protos/ @elastic/security-external-integrations
-/x-pack/packetbeat/ @elastic/security-external-integrations
+### Deploy
+# All changes to the different deployments must be reviewed by data plane team.
+# TODO: This where k8s falls
+/deploy/ @elastic/elastic-agent-data-plane
+
+# Beats Build Specific
+/dev-tools/ @elastic/elastic-agent-data-plane
 
-# Filebeat
-# /filebeat/module/ @elastic/integrations
+# Most docs change are inside each beat / module. For global docs, data plane team should review.
+/docs/ @elastic/elastic-agent-data-plane
+
+### Filebeat
+# Filebeat is owned by the data plan team
+# TODO: CLEANUP, go into more details
+/filebeat @elastic/elastic-agent-data-plane
+/x-pack/filebeat @elastic/elastic-agent-data-plane
+
+# Modules are by defaul owned by the integrations team
+/filebeat/module/ @elastic/integrations
+/x-pack/filebeat/module/ @elastic/integrations
+
+# Infra UI modules
 /filebeat/module/elasticsearch/ @elastic/infra-monitoring-ui
 /filebeat/module/kibana/ @elastic/infra-monitoring-ui
 /filebeat/module/logstash/ @elastic/infra-monitoring-ui
-# /x-pack/filebeat/module/ @elastic/integrations
-# /x-pack/filebeat/module/suricata/ @elastic/security-external-integrations
 
-# Metricbeat
-# /metricbeat/module/ @elastic/integrations
+# Security integrations modules
+/x-pack/filebeat/module/suricata/ @elastic/security-external-integrations
+
+
+### Heartbeat
+# TODO: Is this team still accurate?
+/heartbeat/ @elastic/uptime
+/x-pack/heartbeat/ @elastic/uptime
+
+### Journalbeat
+# Journalbeat is owned by the data plan team
+/journalbeat @elastic/elastic-agent-data-plane
+
+### libbeat
+/libbeat/ @elastic/elastic-agent-data-plane
+
+### licenses
+/licenses/ @elastic/elastic-agent-data-plane
+
+### Metricbeat
+# The core of metricbeat is owned by the data plane team. Bits inside are owned by different teams.
+/metricbeat/ @elastic/elastic-agent-data-plane
+/x-pack/metricbeat/ @elastic/elastic-agent-data-plane
+
+# Modules are owned by the integrations team by default
+/metricbeat/module/ @elastic/integrations
+/x-pack/metricbeat/module/ @elastic/integrations
+
+# Infra UI modules
 /metricbeat/module/elasticsearch/ @elastic/infra-monitoring-ui
 /metricbeat/module/kibana/ @elastic/infra-monitoring-ui
 /metricbeat/module/logstash/ @elastic/infra-monitoring-ui
 /metricbeat/module/beat/ @elastic/infra-monitoring-ui
-# /x-pack/metricbeat/module/ @elastic/integrations
 
-# Heartbeat
-/heartbeat/ @elastic/uptime
+#### monitors.d
+# TODO: Does this belong to synthetics? Why is this in the top dir?
+/monitors.d/ @elastic/uptime
```

Comment:
@andrewvc This kind of looks off on this level.

Reply:
That's not right, we can remove this. That directory only exists in `heartbeat` and `x-pack/heartbeat` which should already be covered.

Reply:
Can you follow up with a PR removing the directory?